tool/tgz/list.go

```diff
@@ -0,0 +1,35 @@
+package tgz
+
+import (
+	"archive/tar"
+	"io"
+	"io/ioutil"
+)
+
+// ListFile list contents in a .tar.gz file.
```

Comment:
> ListFile list contents

=>

> ListFile lists regular files header.

It seems this function only lists the regular file header.

Reply:
Done